src/Simulation/Simulators.Type1.Tests/App.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="xunit.methodDisplay" value="method"/>
+  </appSettings>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/></startup></configuration>
```

Comment:
I am not sure what this file does, but the formatting in it looks off.

Reply:
Hmm... that is odd. It looks like all the simulator tests have this file in the same way, so it must have just been copied over and over again. I could fix them all here to make them more readable.

Reply:
It's a minor issue, so I'm OK with not fixing all of them right now. :)